ci-operator/step-registry/cucushift/installer/check/nutanix/51056/cucushift-installer-check-nutanix-51056-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/check/nutanix/51056/cucushift-installer-check-nutanix-51056-ref.yaml",
```

Comment:
Generally we are not naming the step with case number, some examples for your references:
```
cucushift-installer-check-gcp-disktype-compute
cucushift-installer-check-capability
```